docs/pipelines-terminology.md

```diff
@@ -66,10 +66,18 @@ In the lists and table below, Software Artifacts includes: Documentation Source
 #### Deploy Stage
 * Semantics: Deploy software artificats and documentation to any environment other than the pipeline environment. Verify successful deployment.
 * Aliases: Install
-* Inputs: Software Source, Binary Source, Generated Software, Generated Binaries, Deployment KPIs
+* Inputs: Deployment Configuration, Software Source, Binary Source, Generated Software, Generated Binaries, Deployment KPIs
 * Outputs: Deployment Records, Deployment Reports, Secrets to access deployed resources
 * Other Results and Side Effects: Documentation hosted in a review, staging or production environment. Software running in a review, staging or production environment. Deployment KPIs might influence how the deployment proceeds; for example, for Canary or Blue/Green deployments.
 
+#### Operations Stage
+* Semantics: monitoring of deployed assets
+* Aliases:
```

Comment:
```suggestion
* Aliases: Steady state
```

Reply:
thanks, incorporated this change + some other aliases